cli/src/fmt/simple/payment_path.rs

```diff
@@ -0,0 +1,65 @@
+use super::Simple;
+use fmt::Render;
+use stellar_client::resources::PaymentPath;
+
+impl Render<PaymentPath> for Simple {
+    fn render(&self, payment_path: &PaymentPath) -> Option<String> {
+        let mut buf = String::new();
+        append_to_buffer!(
+            buf,
+            "Destination amount:               {}",
+            payment_path.destination_amount()
+        );
+        append_to_buffer!(
+            buf,
+            "Destination asset code:           {}",
+            payment_path.destination_asset().code()
+        );
+        append_to_buffer!(
+            buf,
+            "Destination asset type:           {}",
+            payment_path.destination_asset().asset_type()
+        );
+        append_to_buffer!(
+            buf,
+            "Destination asset issuer:         {}",
+            payment_path.destination_asset().issuer()
+        );
+        append_to_buffer!(
+            buf,
+            "Source amount:                    {}",
+            payment_path.source_amount()
+        );
+        append_to_buffer!(
+            buf,
+            "Source asset code:                {}",
+            payment_path.source_asset().code()
+        );
+        append_to_buffer!(
+            buf,
+            "Source asset type:                {}",
+            payment_path.source_asset().asset_type()
+        );
+        append_to_buffer!(
+            buf,
+            "Source asset issuer:              {}",
+            payment_path.source_asset().issuer()
+        );
+        append_to_buffer!(buf, "PATH:");
+        for asset in payment_path.path().iter() {
+            append_to_buffer!(buf, "   Path asset code:                  {}", asset.code());
```

Comment:
Looks like you have some leading spaces here to indent the path attributes.  I can't wait to have @alexanderreiff 's pr merged so you can utilize all the awesome tooling he's putting out around this.

Reply:
Dude pumped here too. I'll leave this here for now.